### DISCUSSION TITLE
404 error after running the www package

### DISCUSSION THREAD

**Discussion by karan316** [supabase/supabase#3084] [Questions] [answered]
I am trying to get supabase working locally. I installed the dependencies in the www package. But when I run dev command, http://localhost:3000/ shows me a 404 error page. What am I missing here?

**Comment by soedirgo** [comment, marked as answer]
Hey @karan316, can you try following the steps [here](https://github.com/supabase/supabase/blob/master/development.md)? You may need to run set up Vercel first with `make dev`.

**Reply by karan316** [reply]
Okay, thanks a lot! I was searching for a development guide. Missed this one.

**Reply by karan316** [reply]
Would it be possible to link this file somewhere in the README? Might save some setup time for newcomers 😃

**Reply by soedirgo** [reply]
Will do, thanks!
